tests/asserts/processing/response.py

```diff
@@ -5,11 +5,19 @@
 
 def valid_post(workspace, publ_type, name, response, expected=None):
     expected = expected or {}
-    publ_type_dir = util.get_directory_name_from_publ_type(publ_type)
-    exp_response = {
-        'name': name,
-        'url': f'http://{settings.LAYMAN_PROXY_SERVER_NAME}/rest/workspaces/{workspace}/{publ_type_dir}/{name}'
-    }
+    if isinstance(response, dict) and 'uuid' in response:
+        uuid = response['uuid']
+        publ_type_dir = util.get_directory_name_from_publ_type(publ_type)
+        exp_response = {
+            'name': name,
+            'url': f'http://{settings.LAYMAN_PROXY_SERVER_NAME}/rest/{publ_type_dir}/{uuid}'
+        }
+    else:
```

Comment:
When does this path happen? As `exp_response` is always dict and `url` should always be `f'http://{settings.LAYMAN_PROXY_SERVER_NAME}/rest/{publ_type_dir}/{uuid}'` I can think only of cases, when `response` is `None` and then we should just return true.